css/src/components/form/checkbox.scss

```diff
@@ -15,6 +15,13 @@ $checkbox-timing-function: $input-timing-function !default;
 $checkbox-duration: $input-transition-duration !default;
 $checkbox-spacing: $spacer-3 !default;
 
+$checkbox-danger-border-color: $danger !default;
+$checkbox-success-border-color: $success !default;
+
+$checkbox-hover-border-color: $primary !default;
+$checkbox-focus-border-color: $primary !default;
+$checkbox-focus-box-shadow-size: 0 0 0 0.125rem !default;
```

Comment:
I'm not sure what the best guidance is here, but I find it a little strange that the keyboard focus rect is different between checked and uncheck states. Do we need this?

Reply:
updated